src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -478,7 +477,7 @@ class SharedLinkSection extends React.Component<Props, State> {
                                 />
                             }
                         >
-                            <span className="shared-link-expiration-badge">
+                            <span aria-label="expires" className="shared-link-expiration-badge" role="img">
```

Comment:
`expires` should be localized since this will be read by screen readers. could we add it to the `messages.js` file?

Reply:
oop yes ty